### ISSUE TITLE
No data cells assigned to table

### ISSUE THREAD

**Issue by GaryRidgway** [uiowa/uiowa#9303]
# Problem

If you search for a cohort that has no members with a URL argument, the staff directory will show no results and get flagged for a table with headers but no content.

-[ Here is an example.](https://facilities.uiowa.edu/staff-directory/?cohort=2085113)

# Proposed solution
Detect when the table is going to be empty and do not render it. Instead, render a "no content found for this search criteria" or something similar.

**Comment by briand44** [comment]
Sounds like we thought this should be addressed on the Directory Profiles side of things. Going to close this as not planned for now and make visible to Denny. If it comes back to us, we can re-open.
